src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let sumCallsCount = 0;
+
+  return (a) => (b) => {
```

Comment:
Not the most readable syntax.
```suggestion
  return (a) => {
    return (b) => {
```